wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/Cells/ConversationEncryptionProtocolCell.swift

```diff
@@ -22,7 +22,8 @@ class ConversationEncryptionProtocolCell: DetailsCollectionViewCell {
 
     var encryptionProtocol: String {
         get {
-            return label.text ?? EncryptionProtocol.proteus.rawValue
+            // TODO: [WPB-6205] is this always proteus or can it be influenced by Feature.MLS.Config default protocol?
+            return label.text ?? L10n.Localizable.Conversation.Create.ProtocolSelection.proteus
```

Comment:
But how can a user access this protocol selector? I can not find a way to see the view controller or cells as user